src/components/shared/Footer.jsx

```diff
@@ -1,10 +1,82 @@
+import Link from 'next/link';
 import Container from './Container';
 
 const Footer = () => {
   return (
-    <footer>
+    <footer className='mt-10'>
       <Container>
-        <h1>This is footer component</h1>
+        <div>
+          <div className='flex  flex-col md:flex-row justify-between gap-5 md:gap-0'>
+            {/* row 1 */}
+            <div className='md:w-[40%] space-y-3'>
+              {/* web name */}
+              <h2 className='text-2xl font-extrabold'>FoodiesHub</h2>
+              <p className='font-medium'>
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              </p>
```

Comment:
Bro writes relevant online food order web application content, which should have in the footer.